Add docs for `HTMLLinkElement.integrity` and `HTMLScriptElement.integrity` (#33355)
mdn/content: files/en-us/web/html/element/script/index.md

      - : Default: Signals automatic determination of fetch priority relative to other external scripts.

- `integrity`

>   - : This attribute contains inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. The attribute must not specified when the `src` attribute is not specified. See [Subresource Integrity](/en-US/docs/Web/Security/Subresource_Integrity).

Reviewer: ```suggestion
  - : This attribute contains inline metadata that a user agent can use to verify that a fetched resource has been delivered without unexpected manipulation. The attribute must not specified when the `src` attribute is not specified. See [Subresource Integrity](/en-US/docs/Web/Security/Subresource_Integrity).
```
